### DISCUSSION TITLE
Discrepancy between docker-depoyment-limit & WikiJS SystemInfo

### DISCUSSION THREAD

**Discussion by the-hotmann** [requarks/wiki#6863] [Error / Bug Report] [answered]
I've constrained my container to a 500MB RAM limit, as reported by `docker stats`:

CONTAINER ID | NAME | CPU % | MEM USAGE / LIMIT | MEM % | NET I/O | BLOCK I/O | PIDS
-|-|-|-|-|-|-|-
a5d8104bcceb | wikijs | 0.02% | 136.6MiB / 500MiB | 27.32% | 335kB / 219kB | 4.1kB / 45.1kB | 12

However, the WikiJS internal Systeminfo under Host Information reports different info:

![image](https://github.com/requarks/wiki/assets/17692860/987036d5-34a0-4190-95fa-a268f56d4ac8)

While I understand that the `Total RAM` may refer to the overall server RAM, it could be misleading when running WikiJS in a Docker container. I would prefer if the `Total RAM` reflected the RAM availableto the application/container , which in my case is 500MB, not 12GB.

I'm open for discussion :)

**Comment by NGPixel** [comment, marked as answer]
This is the value as reported by the Node.js process.

**Reply by the-hotmann** [reply]
Thanks, but since I limited down the Memory, this value mist be wrong, or at least highly missleading.

**Reply by NGPixel** [reply]
Then you should file a bug with the node.js team. As previously mentioned, Wiki.js is returning what `os.totalmem()` provides:

https://nodejs.org/api/os.html#ostotalmem

**Reply by the-hotmann** [reply]
Ok thanks, will do so :)

**Reply by the-hotmann** [reply]
https://github.com/nodejs/node/issues/51095
